games-emulation/pcsx2/pcsx2-9999.ebuild

```diff
@@ -0,0 +1,106 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+PLOCALES="ar_SA ca_ES cs_CZ de_DE es_ES fi_FI fr_FR hr_HR hu_HU id_ID it_IT ja_JP ko_KR ms_MY nb_NO pl_PL pt_BR ru_RU sv_SE th_TH tr_TR zh_CN zh_TW"
+
+inherit cmake-utils eutils git-r3 l10n multilib toolchain-funcs wxwidgets
+
+DESCRIPTION="A PlayStation 2 emulator"
+HOMEPAGE="http://www.pcsx2.net"
+EGIT_REPO_URI="git://github.com/PCSX2/pcsx2.git"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS=""
+IUSE="png"
```

Comment:
I smell an automagic dependency. The only place the png USE flag is used is in DEPEND where it pulls in `dev-cpp/pngpp`. That suggests that the build system does something unconditionally when `dev-cpp/pngpp` is present.